VTEX - Logistics API.json

```diff
@@ -3808,14 +3798,16 @@
                                                 "type": "array",
                                                 "description": "List of sales channels associated.",
                                                 "items": {
-                                                    "type": "string"
+                                                    "type": "string",
+                                                    "description": "Sales channel code."
```

Comment:
I suggest:
- Mentioning "trade policy" (official current name for this concept), even if inside parenthesis.
- Linking to related documentation for more information.
- Replacing `code` with `ID`.

There are other fields that already follow these suggestions in this API.